src/pudl/extract/ferc1.py

```diff
@@ -758,10 +758,10 @@ def extract_xbrl(
         return ferc1_raw_dfs
 
     for pudl_table in ferc1_settings.tables:
-        # TODO: Raise exception once XBRL tables are fully integrated
-        # For now skip because map is not defined for all pudl tables
+        if pudl_table not in TABLE_NAME_MAP:
+            raise ValueError(f"{pudl_table} not found in the list of known tables.")
         if "xbrl" not in TABLE_NAME_MAP[pudl_table]:
-            continue
+            raise ValueError(f"No XBRL tables have been associated with {pudl_table}.")
```

Comment:
I could actually imagine cases in which we don't have an XBRL table or a DBF table input for all pudl tables. But i think this is fine for now bc this is our current pattern